pmd-java/src/main/resources/category/java/bestpractices.xml

```diff
@@ -71,7 +71,7 @@ public class Outer {
           class="net.sourceforge.pmd.lang.java.rule.bestpractices.AccessorMethodGenerationRule"
           externalInfoUrl="${pmd.website.baseurl}/pmd_rules_java_bestpractices.html#accessormethodgeneration">
         <description>
-When accessing a private field / method from another class, the Java compiler will generate a accessor methods
+When accessing a private field / method from another class, the Java compiler will generate an accessor methods
```

Comment:
That's plural: methods - should there be no article at all then?
-> ... the Java compiler will generate accessor methods

Or we make it singular
-> ... the Java compiler will generate an accessor method

Reply:
Oh, right.

I think that if it's one accessor method for each field/method then maybe it should be:
"When accessing **a** private field / method from another class, the Java compiler will generate **an** accessor **method**"

If we're talking in general then I guess the plural is better:
"When accessing private **fields / methods** from another class, the Java compiler will generate accessor **methods**"

What do you think?

```suggestion
When accessing private fields / methods from another class, the Java compiler will generate accessor methods
```

Reply:
I like the general case, thanks!